### PR TITLE
add flux update/rollback info

### PR DESCRIPTION
**Pull request by joshgav**
- Add instructions for updates and rollbacks with Flux.
- Add `--token-auth` flag to `flux bootstrap ...` command (hat tip to @yogeek in #115)

### CODE REVIEW

**Review by yogeek — CHANGES_REQUESTED** [review]
- In the `Prerequisites` section, maybe we can add the scope of the "GitHub personal access token" to create ? (something like `that can create repositories by checking all permissions under repo`)

- the command to retrieve the flux git secret fails with some jq versions because of the `@base64d` filter ([issue](https://github.com/stedolan/jq/issues/1408))
```
kubectl get secret podtato-flux-secret -n flux-system -ojson \
    | jq -r '.data."identity.pub" | @base64d'
jq: error (at <stdin>:35): base64d is not a valid format
```
=> maybe we can replace the command by using the `base64` binary which is often already present on system :
```
kubectl get secret podtato-flux-secret -n flux-system -ojson \
    | jq -r '.data."identity.pub"' | base64 -d
```

File: delivery/flux/README.md [hunk 1]
```diff
@@ -277,11 +278,21 @@ xdg-open http://${ADDR}:${PORT}/
...
 changes.
 
-// TODO: describe how to update to a new version by modifying the git repo
+To update the Flux control plane, run the `flux bootstrap ...` command again; it
+will push a new commit to the repo with the latest component configurations.
+
+To update a HelmRelease managed by Flux, make the desired changes to the chart
+and bump the `version` property in Chart.yaml, then commit these changes. For
+example, change the `replicas` value to `2` in `./deliver/chart/values.yaml`,
+bump the `version` value in `./deliver/chart/Chart.yaml`, commit and push the
```
Comment (yogeek) [inline delivery/flux/README.md:287]: typo : `delivery` instead of `deliver` in both path

**Review by yogeek — CHANGES_REQUESTED** [review]

File: delivery/flux/README.md (hunk 1 above)
Comment (joshgav) [reply delivery/flux/README.md]: Fixed!

**Comment by joshgav** [comment]
Thank you for the review @yogeek, I made all those changes.
I'll go ahead and merge this shortly.

**Pull request merged**